### DISCUSSION TITLE
When Retrieving a site it lists me as B1 (Solar) and E1

### DISCUSSION THREAD

**Discussion by craigcurtin-dev** [amberelectric/public-api#36] [General]
Previously i was E1TOU - is this all just rolled into one now ??

Is there a way to get a seperate feed for the solar and actual prices ? It appears to just dump all the solar first and then cost data 2nd ?

Craig

**Comment by madpilot** [comment]
They are all in the same feed - you can filter using the `channelType` attribute: either `general` or `feedIn`

**Reply by craigcurtin-dev** [reply]
OK now worries - some jsonata magic coming up i guess

In relation to the other question - even though it shows the feed as E1 - will that automatically adjust for different people on TOU (as i am) - whereas before it was explicitly broken out ?

Craig

**Reply by madpilot** [reply]
E1 related to the channel on the smart meter, not the tariff (The old API made some weird assumptions - E1 is usually your general meter, which is why it broke it out that way)

When you look at the results from the `/sites` endpoint you will see your channel mapping - the one that is labelled "general" is the one you want.

Any result in the JSON that gets returned from `/prices` that has `channelType: 'general'` will already have the time of use charges included. You can see what time of use period you are in by looking at the `tariffInformation` object.

**Reply by craigcurtin-dev** [reply]
Ok good one thanks- you can close this off then

Craig
